workorder-job-repeater.lua

```diff
@@ -0,0 +1,296 @@
+-- reduces pauses between jobs of infinite work orders
+
+local function print_help()
+    print[====[
+
+workorder-job-repeater
+======================
+This script recreates workshop jobs upon completion in case
+they come from a work order with amount set to infinite. Usually
+a job for an infinite work order is only created once a day,
+resulting in dwarfs lazying around (this doesn't apply to non-infinite
+work orders). This script makes the dwarfes work harder.
+
+For best experience add following to your ``onMapLoad*.init``::
+
+    workorder-job-repeater start
+    
+**Usage**:
+
+``[enable | disable] workorder-job-repeater``
+
+``workorder-job-repeater start [<frequency>] | stop``
+
+``workorder-job-repeater --frequency <ticks> | --verbose | --help``
+
+:enable, start <frequency>: starts job monitoring, running every ``<frequency>`` ticks (default: 10).
```

Comment:
I think you should pick a paradigm and stick with it. Enable/disable or start/stop, not both. Enable/disable before or after the command, not both.

Reply:
kept only enable/disable after command